### ISSUE TITLE
generate output image

### ISSUE THREAD

**Issue by imanrekh** [lisjin/dcan-tensorflow#11]
hi i trained the network successfully, and i have done test by using "evaluate" but i need to generate output images for test set. i am going to regenrate the images that are shown in "DCAN: Deep Contour-Aware Networks for Accurate Gland Segmentation", is there any method in your implementation to regenerate the images.
i will be appreciate for your response.

**Comment by lisjin** [comment]
Sorry, I don't have any code for post-processing the segments and contours. Maybe [this](https://www.mathworks.com/help/images/marker-controlled-watershed-segmentation.html) article could give you some clues.

**Comment by imanrekh** [comment]
I did not mean post-processing. I need to save network output as image, do you have any function in your code to save results as image?

**Comment by lisjin** [comment]
I just used TensorBoard for viewing the images. The [`dice_op`](https://github.com/lisjin/dcan-tensorflow/blob/9e44ac1a1810655a0b0a5cb4b999654ce799b5fe/tf-dcan/bbbc006.py#L457) function called during evaluation calls `get_show_preds` and `get_show_labels` in TensorBoard. If you wanted to save the images, you could follow [this ](https://stackoverflow.com/a/51005985) solution.
